pkg/agent/openflow/client.go

```diff
@@ -584,6 +597,13 @@ func (c *client) installMulticastPodMetricFlows(interfaceName string, podIP net.
 func (c *client) UninstallPodFlows(interfaceName string) error {
 	c.replayMutex.RLock()
 	defer c.replayMutex.RUnlock()
+	if c.enableMulticluster {
+		// If multi-cluster is enabled, we store podClassifierFlow with a different key in
```

Comment:
Multi-cluster

Reply:
This code block has been removed. Mark as resolved.